docs/advanced.md

```diff
@@ -130,7 +130,7 @@ If you wish to bypass the generation of exit codes 2 and 3, you may do so with t
 
 ## Target Versions
 
-By default, Pluto was designed with deprecations related to Kubernetes v1.16.0. However, as more deprecations are introduced, we will try to keep it updated.
+By default, Pluto was designed with deprecations related to Kubernetes v1.22.0. However, as more deprecations are introduced, we will try to keep it updated.
```

Comment:
```suggestion
Pluto was originally designed with deprecations related to Kubernetes v1.16.0.  As more deprecations are introduced, we will try to keep it updated. Community contributions are welcome in this area.

Currently, Pluto defaults to a targetVersion of v1.22.0, however this is configurable (please continue reading)
```

Reply:
The original intent was to tell the history of Pluto. Agreed that it is now confusing. I suggested some alternative wording here. Thanks!

Reply:
@sudermanjr 
The update that you suggested are easy to understand and we want to know history about Pluto :) Thank you very much. I added your suggested commit.